corehq/pillows/management/commands/utils.py

```diff
@@ -0,0 +1,126 @@
+import json
+import sys
+from collections import OrderedDict
+
+
+def pprint(obj, namespace={}, stream=sys.stdout, indent=4, separators=(',', ': ')):
+    encoder = ElasticMappingEncoder(namespace, indent=indent, separators=separators)
+    for chunk in encoder.iterencode(obj):
+        stream.write(chunk)
+
+
+class ElasticMappingEncoder(json.JSONEncoder):
```

Comment:
docstring explaining why this is needed would be great

Reply:
Docstrings added (new commit).